docs/configuration/http_filters/fault_filter.rst

```diff
@@ -66,6 +68,20 @@ upstream_cluster:
   the config are present in the request with the same values (or based on presence if the ``value``
   field is not in the config).
 
+.. _config_http_filters_fault_injection_downstream_cluster:
+
+downstream_cluster_specific_settings:
```

Comment:
The defaults for the lookups for the downstream specific cluster runtime lookups can be taken from the main config. So if they aren't present, the behavior will be the same. I really don't see people wanting to do 2 runtime lookups to get a default out of runtime, then override with a cluster specific one. So I think we can collapse this logic and make it simpler. Let's go ahead and do that.